torch_xla/csrc/module.cpp

```diff
@@ -25,12 +25,16 @@
 namespace torch_xla {
 namespace {
 
-void GatherParameters(std::vector<at::Tensor*>* values,
+void GatherParameters(std::vector<at::Tensor>* values,
                       std::vector<bool>* requires_grad,
                       const torch::jit::script::Module& m) {
   for (auto& param : m.get_parameters()) {
-    values->push_back(param->slot());
-    requires_grad->push_back(!param->is_buffer);
+    values->push_back(param->slot()->toTensor());
+    requires_grad->push_back(true);
+  }
+  for (auto& param : m.get_attributes()) {
```

Comment:
Attributes can be any type, so you should do a `param->slot()->isTensor()` check

Reply:
Just for more context `get_parameters()` now only returns `nn.Parameters` (so everything that is in here is not a buffer). Things that were buffers before are now in a separate list we're calling `attributes`, which can be accessed by `get_attributes()`. However, this can also have any random module attribute, like an `int` or a `Dict`, so you need to type check things that come out of here.

Reply:
Done.